core/templates/services/playthrough.service.ts

```diff
@@ -104,7 +105,8 @@ class CyclicStateTransitionsTracker {
 
   private makeCycle(collisionIndex: number): string[] {
     const collision = this.pathOfVisitedStates[collisionIndex];
-    const cycleWithNoCollision = this.pathOfVisitedStates.slice(collisionIndex);
+    const cycleWithNoCollision = (
```

Comment:
Why is this changed?

Reply:
Due to some line length problem in eslint .